engine/runtime/src/main/java/org/enso/interpreter/node/expression/builtin/meta/TypeOfBuiltin.java

```diff
@@ -13,12 +16,20 @@
     autoRegister = false)
 @GenerateUncached
 public final class TypeOfBuiltin extends Node {
+  private @Child WarningsLibrary warnings = WarningsLibrary.getFactory().createDispatched(11);
   private @Child TypeOfNode typeOf = TypeOfNode.create();
 
   private TypeOfBuiltin() {}
 
-  public Object execute(@AcceptsError Object value) {
-    return typeOf.execute(value);
+  public Object execute(@AcceptsError @AcceptsWarning Object value) {
```

Comment:
Since 12ed8a3b23786574e0fe692b1c9ccac272cf651e the type of a _value with a warning_ shouldn't be annotated by a warning. That's the behavior we want, right?

The default behavior of _builtin nodes_ like `TypeOfBuiltin` is to remove warnings from the `value` argument (if there are any) and re-attach them when the result is computed.

The result of this builtin is `Type` and I believe types should be _meta objects_ - e.g. be the shared representation, independent of the individual values. Hence it **doesn't matter if the value has a warning** attached - the type (even it is computed from the value) isn't influenced by the warning associated with the value.

By `@AcceptsWarning` and not attaching it back to the result of this builtin - the `Type` stays as it is - e.g. not annotated.